nucliadb/src/nucliadb/export_import/datamanager.py

```diff
@@ -58,20 +59,33 @@ async def get_metadata(self, type: str, kbid: str, id: str) -> Metadata:
         if data is None or data == b"":
             raise MetadataNotFound()
         decoded = data.decode("utf-8")
-        model_type = {
-            "export": ExportMetadata,
-            "import": ImportMetadata,
-        }[type]
-        return model_type.parse_raw(decoded)  # type: ignore
+        if type == "export":
+            model_type = ExportMetadata
+        elif type == "import":
+            model_type = ImportMetadata  # type: ignore
+        else:
+            raise ValueError(f"Invalid type: {type}")
+        json_decoded = json.loads(decoded)
+
+        # For some reason, the total and processed fields are not always present in the metadata.
+        # This is to unblock already created exports that hit this bug.
+        if json_decoded.get("total") is None:
+            json_decoded["total"] = 0
+        if json_decoded.get("processed") is None:
+            json_decoded["processed"] = 0
```

Comment:
NIT: you could use `setdefault` here
```suggestion
        json_decoded.setdefault("total", 0)
        json_decoded.setdefault("processed", 0)
```

Reply:
The problem is that the keys are present but they come set to `None`